panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp

```diff
@@ -222,11 +257,31 @@
         }
     };
 
+    let clearInputFieldsAndResetURL = function(searchTabName) {
+        switch (searchTabName) {
+            case "experiment":
+                clearInputFromExperimentTab();
+                resetUrl();
```

Comment:
Can move `resetUrl()` outside of the `switch` since it's identical across all branches.